oneqa/boolqa/processors/postprocessors/extractive.py

```diff
@@ -0,0 +1,40 @@
+from oneqa.mrc.processors.postprocessors.extractive import ExtractivePostProcessor
+from oneqa.mrc.data_models.eval_prediction_with_processing import EvalPredictionWithProcessing
+
+from datasets import Dataset
+from tqdm import tqdm
+
+from typing import List, Dict, Any, Tuple
+
+class ExtractivePipelinePostProcessor(ExtractivePostProcessor):
+    
```

Comment:
Suggest to add doc string to this class and other classes in pre/post processors.